docs/installation/_index.md

```diff
@@ -8,3 +8,46 @@ Headlamp can be deployed in a Kubernetes cluster, or run as a desktop applicatio
 
 Please check the guides in this section for installing the desktop application
 or deploying Headlamp in your cluster.
+
+## Authentication / Log-in
+
+Currently you can log in Headlamp by using a **client-certificate** (as you may have configured with e.g. minikube), or a **bearer token**.
+
+Headlamp uses [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac) for checking whether and how users can access resources. This means that the
+recommended way to log in into Headlamp is to use a Service Account token.
+
+### Create a Service Account token
+
+As an example, you can create a service account for using Headlamp and retrieve its token to
+authenticate:
+
+1. Create a Service Account:
+
+```shell
+kubectl create serviceaccount headlamp-admin
+```
+
+2. Give admin rights to the account (or check the RBAC docs on the Service Account
+[use](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#service-account-permissions)):
```

Comment:
Not sure what you mean by the sentence in parenthesis. I'd mention users can check the RBAC docs if they want to set more restrictive permissions (and assume they're advanced enough to figure it out).

```suggestion
2. Give admin rights to the account (check the [RBAC docs](https://kubernetes.io/docs/reference/access-authn-authz/rbac) if you want to set more restrictive permissions):
```